### PR TITLE
Add brightness and password functionality

### PR DESCRIPTION
**Pull request by tekka007**
Add methods to set screen brightness (set_screen_brightness) and password (set_password)

### CODE REVIEW

File: core/idotmatrix/common.py [hunk 1]
```diff
@@ -101,7 +123,7 @@ def setJoint(self, mode):
         Returns:
             _type_: byte array of the command which needs to be sent to the device
         """
-        try:
+        try:
```
Comment (derkalle4) [inline core/idotmatrix/common.py:126]: Could you please clean this up? Seems like you have pressed the TAB key.

File: core/idotmatrix/common.py [hunk 2]
```diff
@@ -113,3 +135,33 @@ def setJoint(self, mode):
             )
         except BaseException as error:
             logging.error("could not change the device joint: {}".format(error))
+            
+    def set_password(self, password: int) -> None:
```
Comment (derkalle4) [inline core/idotmatrix/common.py:139]: Please split the set password stuff into a separate pull request as this is a separate feature :)

File: core/cmd.py [hunk 3]
```diff
@@ -255,6 +271,30 @@ async def togglescreen(self):
...
+        """sets connection password"""
+        try:
+            conv_password = int(argument)
+            if len(argument) == 6 and conv_password in range(0, 1000000):
+                self.logging.info(f"setting password: {argument}")
+                await self.bluetooth.send(Common().set_password(conv_password))
+            else:
+                self.logging.error(f"Password should be 6 digits long and in range 000000...999999")
+        except ValueError:
+            self.logging.error(f"Invalid integer: {argument}")
```
Comment (derkalle4) [inline core/cmd.py:296]: We should either agree on using

`f"{argument}"`

or 

`"{}".format()`

but not both.

I really like **.format** because it provides a better readability at least for me because when using multiple aguments with longer names one could simply split them into multiple lines with ease.

Anyhow, I am fine with both but not mixed up.

File: core/idotmatrix/common.py [hunk 4]
```diff
@@ -36,6 +36,28 @@ def rotate180degrees(self, type=0):
         except BaseException as error:
             logging.error("could not rotate the screen of the device: {}".format(error))
 
+    def set_screen_brightness(self, brightness_percent: int) -> None:
```
Comment (derkalle4) [inline core/idotmatrix/common.py:39]: Nice catch with the type casting.. have missed that for the initial version. Will add this to all functions later on :)

**Review by derkalle4 — CHANGES_REQUESTED** [review]
Many thanks for your commit! I highly appreciate your work and your interest in this little project! I have some suggestions for you. Maybe we can discuss them and agree on :)

Kind Regards,
Kalle

File: core/cmd.py (hunk 3 above)
Comment (tekka007) [reply core/cmd.py:296]: Disagree - the former provides a better readibility IMO, especially on a multiline statement.

```
print(f"var1 = {var1}, var2 = {var2} and the sum is {var1 + var2}"

# vs

print("var1 = {}, var2 = {} and the sum is {}".format(var1, var2, var1+var2))
```

File: core/idotmatrix/common.py (hunk 4 above)
Comment (tekka007) [reply core/idotmatrix/common.py:39]: Yeah, type casting would be good practice and simplify the code :)

File: core/idotmatrix/common.py (hunk 1 above)
Comment (tekka007) [reply core/idotmatrix/common.py]: Done

File: core/idotmatrix/common.py (hunk 2 above)
Comment (tekka007) [reply core/idotmatrix/common.py:139]: For larger PRs certainly - however, at this stage of the project I'd rather value clear commit messages over single feature per branch principles (probably a matter of taste :) ). lmk.

File: core/cmd.py (hunk 3 above)
Comment (derkalle4) [reply core/cmd.py:296]: Okay, fair enough! We'll keep it and maybe I have to change my code style then :)

**Comment by derkalle4** [comment]
Merged your request! Many thanks for helping out!
